### DISCUSSION TITLE
How do I'm make tiles generate in the world + recipes

### DISCUSSION THREAD

**Discussion by PHITHGOD24** [MinicraftPlus/minicraft-plus-revived#709] [Q&A] [answered]
### HIII
I'm wanna make a new type of ore called emerald generate in the underground, I'm still new to modding the game

_oh ya I'm also wanna know how to add recipes to the game_

**Comment by PHITHGOD24** [comment]
hello?

**Comment by BenCheung0422** [comment, marked as answer]
Actually if you want to directly mod the game, I would recommend you to look through the codebase and understand the code, before making changes. Otherwise I would recommend you trying out [mod loader](https://minicraft.wiki.gg/wiki/Mod_loader) like [MiniFabric](https://minicraft.wiki.gg/wiki/MiniFabric) just for modding, though its development has been inactive for a significant long time.

**Comment by PHITHGOD24** [comment]
ok
